Exercise03-DataTypes/Hristomir-Hristov/DTYPES01-006.js

```diff
@@ -0,0 +1,19 @@
+/*
+
+Define four string variables with your three names (first, middle and family).
+The fourth variable initialize with all names, separated with a tab character. Print on the
+console the length of each variable.
+
+*/
+
+let firstName = 'Hristomir';
+let middleName = "Ilkov";
+let lastName = 'Hristov';
+let allNames = `${firstName}    ${middleName}   ${lastName}`;
+
+console.log(firstName.length);
+console.log(middleName.length);
+console.log(lastName.length);
+console.log(allNames.length);
+
+// length of variable different from length of string ??
```

Comment:
In that particular case is basically the same thing. The length of a variable is equal to the length of the data that it holds, except for numbers(integers and doubles) and boleans. If you call the `length` method on a number/boolean, it will print `undefined`.